onnxscript/tests/function_libs/torch_aten/ops_correctness_test.py

```diff
@@ -1502,14 +1535,14 @@ def _capture_graph_and_evaluate_torch_script_evaluator(function: Callable, args,
 
         onnx_model = onnxscript_graph.to_model_proto(TEST_OPSET_VERSION)
         # Make sure the model is valid
-        try:
-            onnx.checker.check_model(onnx_model, full_check=True)
-        except onnx.checker.ValidationError as e:
-            raise AssertionError(
-                f"ONNX model is invalid: {e}. "
-                f"Model:\n"
-                f"{onnxscript.proto2text(onnx_model)}"
-            ) from e
+        # try:
+        #     onnx.checker.check_model(onnx_model, full_check=True)
+        # except onnx.checker.ValidationError as e:
+        #     raise AssertionError(
+        #         f"ONNX model is invalid: {e}. "
+        #         f"Model:\n"
+        #         f"{onnxscript.proto2text(onnx_model)}"
+        #     ) from e
```

Comment:
Once the tests pass, I would uncomment this, mark xfails for the tests for now, and submit an issue to onnx. Hopefully we can get the fix in a newer weekly release then enable the xfail'ed tests again.

Reply:
FullGraph mode can passed without check_model. So marked as xfails.